Document/0x05b-Basic-Security_Testing.md

```diff
@@ -107,7 +107,7 @@ For a typical mobile app security build, you'll usually want to test a debug bui
 
 #### Recommended Tools
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1227>
+This section describes some of the most popular and useful tools for testing security of Android applications. Following below is the aforementioned list of tools, each tool with an introduction, references to the installation instructions, example use cases and the reference to the full list of commands.
```

Comment:
- This section describes - > introduces ?
- each tool with an introduction, -> For each of them we give a brief description ?
- "reference to the full list of commands" ->  + or to the corresponding documentation ? 
- I'd mention that are also tools that we'll be using throughout the guide
- also refer to the "Testing Tools" appendix at the end of this book for a complete list.

Reply:
thanks for the addition @cldrn !

Reply:
Thanks for the feedback. Reworded things and added the reference to the Tools list.